echopype/tests/calibrate/test_calibrate_ek80.py

```diff
@@ -251,4 +251,19 @@ def test_ek80_BB_power_echoview(ek80_path):
     ev_vals = df_real.values[:, :]
     ep_vals = pc_mean.values.real[:, :]
     assert np.allclose(ev_vals[:, 69:8284], ep_vals[:, 69:], atol=1e-4)
-    assert np.allclose(ev_vals[:, 90:8284], ep_vals[:, 90:], atol=1e-5)
\ No newline at end of file
+    assert np.allclose(ev_vals[:, 90:8284], ep_vals[:, 90:], atol=1e-5)
+
+
+def test_ek80_CW_complex_Sv_receiver_sampling_freq(ek80_path):
+    ek80_raw_path = str(ek80_path.joinpath("D20230804-T083032.raw"))
+    ed = ep.open_raw(ek80_raw_path, sonar_model="EK80")
+
```

Comment:
I suggest adding the following here to make it clear what gets changed in `compute_Sv` later.
```suggestion
# Parsed receiver_sampling_frequency is 0
assert ed["Vendor_specific"]["receiver_sampling_frequency"] == 0
```